backend/src/main/java/com/exadel/carinsurance/service/implementation/PhotosService.java

```diff
@@ -55,4 +63,50 @@ public PhotosResponseEntity savePhotos( Long userId, Long assignmentId, List<Mul
         .namesOfPhotosOfImpact( String.join( ";", namesOfPhotosOfImpactList ) )
         .build();
   }
+
+  @Override
+  public ResponseEntity getPhoto(
+      @PathVariable( "userId" ) Long userId,
+      @PathVariable( "assignmentId" ) Long assignmentId,
+      @PathVariable( "photoName" ) String photoName

```

Comment:
why does a service uses a controller level annotations? What is the point in those?

Reply:
Removed it. Done